File: DSA/SakshamDwivedi_590013147/Day16/D16P3.c
#include <stdio.h>

float arr[100];
int size;
float max;

int main(){
    printf("Enter the size of the array : ");
    scanf("%d", &size);
    printf("Enter the elements : ");
    for(int i = 0; i < size; i++){
        scanf("%f", &arr[i]);
    }

    max = arr[0];
    for(int i = 0; i < size; i++){
        if (arr[i] > max){
            max = arr[i];
        }
    }

    if(max == (int)max){
        printf("Max : %d", (int)max);
    }
    else{
        printf("Max : %f", max);
    }
}

File: DSA/SakshamDwivedi_590013147/Day16/D16P3.py
#Solution Function
def max_element(arr):
    if len(arr) < 1:
        return -1
    mel = arr[0]
    for i in range(len(arr)):
        if arr[i] > mel:
            mel = arr[i]
    return mel

#Fixed Input
array = [3, 7, 1, 9, 4, 6]
print(max_element(array))

#Custom Input
array = list(float(x) for x in input("Enter the array : ").split())
print(max_element(array))
